src/pages/en/reference/configuration-reference.mdx

```diff
@@ -546,8 +546,21 @@ Pass additional configuration options to Vite. Useful when Astro doesn't support
 
 View the full `vite` configuration object documentation on [vitejs.dev](https://vitejs.dev/config/).
 
+A common use case is to pass a package name to [`vite.ssr.noExternal`](https://vitejs.dev/config/ssr-options.html#ssr-noexternal) to prevent it from being externalized.
```

Comment:
Do we have a better way to explain/describe what “externalized” means? The code comment about packages being “transformed for SSR” would already be better I think.

Reply:
i figured the code comment plus the link to vite docs would be sufficient

what about this:
```suggestion
A common use case is to pass a package name to [`vite.ssr.noExternal`](https://vitejs.dev/config/ssr-options.html#ssr-noexternal) to prevent it from being externalized from Vite's SSR transform system.
```
or another link
```suggestion
A common use case is to pass a package name to [`vite.ssr.noExternal`](https://vitejs.dev/config/ssr-options.html#ssr-noexternal) to prevent it from being [externalized](https://vitejs.dev/guide/ssr.html#ssr-externals).
```

Reply:
+1 for the second choice. The "externalized" concept is a bit hard to explain in brief so an external link seems to be the best alternative for me.

I also think that maybe we can remove this paragraph, and instead put the link and brief reason in the example's comment instead?

Reply:
Links in code comments aren’t clickable so my preference would be to keep them in a paragraph. You both seem to like the second option, so let’s go with that. Thanks!